shopinvader_delivery_carrier/tests/test_carrier.py

```diff
@@ -6,9 +6,9 @@
 from odoo.addons.shopinvader.tests.test_cart import CommonConnectedCartCase
 
 
-class CarrierCase(CommonConnectedCartCase):
+class CommonCarrierCase(CommonConnectedCartCase):
     def setUp(self):
-        super(CarrierCase, self).setUp()
+        super(CommonCarrierCase, self).setUp()
```

Comment:
@sebastienbeau I would prefer to have this common setup into a common.py file

Reply:
done